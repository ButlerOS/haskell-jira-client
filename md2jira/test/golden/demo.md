---
title: Team backlog
---

Document usage:
- test

# Develop app  {#PROJ-42}

The goal of app is ...

```bash
$ command
output
```

## Implement X

The goal of this story is ...

- [ ] do x {.n}
- [ ] and y

what about:

- [ ] z task

## Implement Z

- [x] completed {.tc}
  > https://localhost

- [ ] todo
