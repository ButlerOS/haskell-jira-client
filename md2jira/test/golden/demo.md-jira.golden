EpicTitle: Develop app
EpicBody:
The goal of app is ...

{code:bash}
$ command
output
{code}

StoryTitle: Implement X
StoryBody:
The goal of this story is ...

* ☐ do x
* ☐ and y

what about:

* ☐ z task



StoryTitle: Implement Z
StoryBody:
* ☒ completed
bq. https://localhost
* ☐ todo


